integration/gethnetwork/main/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/obscuronet/obscuro-playground/integration/gethnetwork"
+)
+
+// Spins up a new Geth network.
+func main() {
+	config := ParseCLIArgs()
+
+	gethBinaryPath, err := gethnetwork.EnsureBinariesExist(gethnetwork.LatestVersion)
+	if err != nil {
+		panic(err)
+	}
+	gethnetwork.NewGethNetwork(config.StartPort, gethBinaryPath, config.NumNodes, 1, config.PrefundedAddrs)
+	fmt.Println("Geth network started.")
```

Comment:
Log maybe ?

Reply:
I did this on purpose so there's some sign of life when things are pushed to the logs. Otherwise, you can think the process is blocked/dead, but you just haven't checked the logs.